### ISSUE TITLE
aria-allowed-attr ignore aria-multiline=false on element with contenteditable 

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4463]
We've seen examples where devs have put `aria-multinline=false` on elements with `contenteditable`, with a role that doesn't support them. The reason for this turns out to be that Safari will otherwise announce that element as being a multinline element. We've seen this on an element with `role=combobox`, but it would be good to test whether this behavior happens with other roles too.

At least the following should pass axe-core:

```html
<!-- This passes -->
<div aria-multiline="false" tabindex="0" aria-labelledby="foo" role="combobox" contenteditable>
```

**Comment by straker** [comment]
If there are more than one not allowed attr and some are falsey and some are violations, we'll ignore the incomplete cases as to not report on false positives. Once the violations are resolved and there are no more to report on for the node, we'll report on the incomplete cases.
